links-slackbot/src/test/java/juja/microservices/links/slackbot/util/SlackTextHandlerTest.java

```diff
@@ -1,7 +1,46 @@
 package juja.microservices.links.slackbot.util;
 
+import org.hamcrest.collection.IsIterableContainingInAnyOrder;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
 /**
- * @author Ivan Shapovalov
+ * @author Ben Novikov
  */
 public class SlackTextHandlerTest {
+    private final SlackTextHandler slackTextHandler = new SlackTextHandler();
```

Comment:
Good practice is to initialize class fields in @Before setUp() method. 
http://projsolution.com/a106-193959-java